16-synthesis.Rmd

```diff
@@ -124,7 +124,7 @@ These resources include @zuur_mixed_2009, @zuur_beginners_2017 which focus on ec
 [*R for Geographic Data Science*](https://sdesabbata.github.io/r-for-geographic-data-science/) provides an introduction to R for geographic data science and modelling.
 
 We have largely omitted geocomputation on 'big data' by which we mean datasets that do not fit on a high-spec laptop.
-This decision is justified by the fact that the majority of geographic datasets that are needed for common research or policy applications *do* fit on consumer hardware.
+This decision is justified by the fact (that aside from large-scale fine-resolution remote sensing data) the majority of geographic datasets that are needed for common research or policy applications *do* fit on consumer hardware.
```

Comment:
Just because we omit largely remote sensing data does not mean that it is also commonly neglected for geographic data analysis -:) Think e.g., of **stars** and OpenEO.

Reply:
Agreed. Minor tweak I suggest is to simplify the adjectives and remove the brackets:

```
This decision is justified by the fact that the majority of geographic datasets that are needed for common research or policy applications *do* fit on consumer hardware, large high resolution remote sensing datasets being a notable exception.
```

Reply:
Excellent, I'll write it like this.